localstack/utils/docker_utils.py

```diff
@@ -770,6 +793,53 @@ def inspect_network(self, network_name: str) -> Dict[str, Union[Dict, str]]:
         except NoSuchObject as e:
             raise NoSuchNetwork(network_name=e.object_id)
 
+    def connect_container_to_network(
+        self, network_name: str, container_name_or_id: str, aliases: Optional[List] = None
+    ) -> None:
+        LOG.debug(
+            "Connecting container '%s' to network '%s' with aliases '%s'",
+            container_name_or_id,
+            network_name,
+            aliases,
+        )
+        cmd = self._docker_cmd()
+        cmd += ["network", "connect"]
+        if aliases:
+            cmd += ["--alias", ",".join(aliases)]
+        cmd += [network_name, container_name_or_id]
+        try:
+            safe_run(cmd)
```

Comment:
nit: Not sure if this is idempotent - getting a message like `endpoint with name <container> already exists in network <network>` when re-attaching a container to one of its existing networks locally. (Probably an edge case - no need to update the PR, but could be considered in a future iteration..)